site_root/input/ImplementationGuide-NCPI-Project-Forge.json

```diff
@@ -141,6 +141,102 @@
         "reference": {
           "reference": "StructureDefinition/age-at-event"
         }
+      },
+      {
+        "description": "NCPI Project Forge StructureDefinition ncpi-document-reference, Base: DocumentReference",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge StructureDefinition/ncpi-document-reference",
+        "reference": {
+          "reference": "StructureDefinition/ncpi-document-reference"
+        }
+      },
+      {
+        "description": "NCPI Project Forge StructureDefinition drs-attachment, Base: Attachment",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge StructureDefinition/drs-attachment",
+        "reference": {
+          "reference": "StructureDefinition/drs-attachment"
+        }
+      },
+      {
+        "description": "NCPI Project Forge ResearchSubject example-subject-id",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge ResearchSubject/example-subject-id",
+        "reference": {
+          "reference": "ResearchSubject/example-subject-id"
+        }
+      },
+      {
+        "description": "NCPI Project Forge DocumentReference example-research-study-document-reference-id",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge DocumentReference/example-research-study-document-reference-id",
+        "reference": {
+          "reference": "DocumentReference/example-research-study-document-reference-id"
+        }
+      },
+      {
+        "description": "NCPI Project Forge Task example-research-study-task-id",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge Task/example-research-study-task-id",
+        "reference": {
+          "reference": "Task/example-research-study-task-id"
+        }
+      },
+      {
+        "description": "NCPI Project Forge ResearchStudy example-project-id",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge ResearchStudy/example-project-id",
+        "reference": {
+          "reference": "ResearchStudy/example-project-id"
+        }
+      },
+      {
+        "description": "NCPI Project Forge Practitioner example-principle-investigator",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge Practitioner/example-principle-investigator",
+        "reference": {
+          "reference": "Practitioner/example-principle-investigator"
+        }
+      },
+      {
+        "description": "NCPI Project Forge Patient example-patient-id",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge Patient/example-patient-id",
+        "reference": {
+          "reference": "Patient/example-patient-id"
+        }
+      },
```

Comment:
Why don't we use the existing Patient example: https://github.com/ncpi-fhir/ncpi-model-forge/blob/master/site_root/input/resources/examples/Patient-pt-001.json

Reply:
Will do

Reply:
resolved - implemented.

Reply:
This IG config file seems to still have the old resource entries. Please run `fhirutil ./site_root/input/resources` again, then this IG config file will also be updated. Then, please re-push this file.

Reply:
resolved.